tool/jt.rb

```diff
@@ -1134,7 +1134,8 @@ def test_specs(command, *args)
 
       options += %w[--excl-tag graalvm]
       options << '-t' << ENV['AOT_BIN']
-      options << '-T-XX:OldGenerationSize=2G'
+      options << '-T-XX:YoungGenerationSize=2G'
+      options << '-T-XX:OldGenerationSize=4G'
```

Comment:
I don't think it matters too much one way or another. If we have memory constraints somewhere, we can lower it.